packages/front-end/components/Experiment/ResultsTable.tsx

```diff
@@ -109,7 +109,7 @@ export default function ResultsTable({
                   }}
                   options={variations.map((v, i) => ({
                     value: i + "",
-                    label: `${i}: ${v.name}`,
+                    label: `${v.id}: ${v.name}`,
```

Comment:
not sure this is visually safe, as "id" is a string and could be something quite long